### ISSUE TITLE
Apostrophe 2, Dyad 2, Baskerville 2 (AT): Sticky posts creating duplicate featured posts

### ISSUE THREAD

**Issue by juliasydnor** [Automattic/themes#170]
## Steps to replicate

1. Activate Apostrophe 2 on an AT site.
2. Publish at least 11 posts.
3. Make at least one post a sticky post.
4. View the site's front page and click "Older posts" to view the next paginated archive page.

## Result

The sticky post(s) appears at the top of the front page, as expected.

<img width="766" alt="front-page-sticky-post" src="https://user-images.githubusercontent.com/29981758/42049772-e89dd2fc-7afe-11e8-908a-74b155e71604.png">

The sticky post is duplicated on the paginated archive page. The sticky class has been removed from the duplicate post, which breaks the "featured post/date" styling in the Apostrophe 2 theme.

<img width="757" alt="paginated-sticky-post" src="https://user-images.githubusercontent.com/29981758/42049857-35ad5568-7aff-11e8-8305-e526e388832b.png"> 

This was also replicated with the Dyad 2 and Baskerville 2 themes on AT sites.

This does not seem to happen on simple sites because of the way older posts are loaded via AJAX.

## Expected

I expected each post to appear only once, so I did not expect the sticky post to be duplicated on the paginated archive page. I expected all posts, including sticky posts, to have the correct styling.

* User report: #29943-hc
* Follow-up ticket: #1218304-zen and #1242936-zen
* User's site: https://funchalnoticias.net/ - the duplicated post with incorrect styling can be seen at https://funchalnoticias.net/page/2/
* Replicated here: https://selaattestsite.blog/ - the duplicated post with incorrect styling can be seen at https://selaattestsite.blog/page/2/
* Possible related Jetpack issue: https://github.com/Automattic/jetpack/issues/2832
* AT request: https://atomicrequests.wordpress.com/2018/06/12/this-one-has-me-a/

**Comment by laurelfulford** [comment]
I can see this issue on Apostrophe 2 and Baskerville 2, but not Dyad 2.

I could also recreate it using the click-to-scroll option for infinite scroll -- so I'm not entirely sure why it's happening on AT sites, but not otherwise.

**Comment by laurelfulford** [comment]
Apostrophe 2 is fixed r49012-wpcom-themes and Baskerville 2 is fixed in r49013-wpcom-themes. These fixes will show up early next week on the users sites.

@juliasydnor Can you please confirm that you're still seeing this issue in Dyad 2? I might have just missed something in recreating it there.

I'm also not 100% sure this is purely a theme issue, or if there's something else at play -- might need more digging if it comes up again!

**Comment by juliasydnor** [comment]
@kristarella @sarahcada Do you happen to still have the site where you tested this on Dyad 2?

**Comment by kristarella** [comment]
I tested it on https://kristarellaatomic20170913.blog

I'll double check settings shortly to see if I can repro again

**Comment by kristarella** [comment]
Sticky post is only repeated in Dyad 2 when infinite scroll in Jetpack is set to "use theme's default behaviour". This is the normal WP behaviour, and it looks fine in Dyad 2, so I don't think there is a problem with that theme. The post is not repeated when using infinite scroll or "load more with button", from what I can see. I can't remember if we found a problem in Dyad, or if we just used it to compare the behaviour that sticky posts show on /page/2 or later.

**Comment by laurelfulford** [comment]
Thanks for confirming, @kristarella! 

A little late, but I've confirmed that the fix has been deployed to the sites in the original report.

@juliasydnor could you please follow up with the user who reported this issue? The ZD ticket is closed, so I can't leave a comment there. Thanks!

**Comment by DrDRRobinson22** [comment]
I have been trying to remove the featured/sticky designation or move it to another article in Apoistrophe 2
